src/components/ProgressIndicator/ProgressIndicator.jsx

```diff
@@ -105,10 +109,13 @@ const ProgressIndicator = ({
   ]);
   // Only recalculate current step if inputs change
   const currentStep = matchingIndex > -1 ? matchingIndex : 0;
-
   return (
     <StyledProgressIndicator
-      className={[className, isVerticalMode ? 'bx--progress--vertical' : ''].join(' ')}
+      className={[
+        className,
+        isVerticalMode ? 'bx--progress--vertical' : '',
+        `${iotPrefix}--progress-indicator`,
+      ].join(' ')}
```

Comment:
The `bx` prefix can't be hardcoded as it's configurable. IIRC we can import it from `settings`. 

Also lets push for consistency and use `classnames` in lieu of `.join(' ')`:

```suggestion
      className={classNames({
        className,
        `${iotPrefix}--progress-indicator`
        `${prefix}--progress--vertical`: isVerticalMode,
      })}
```